src/content-tags-drawer/data/apiHooks.jsx

```diff
@@ -142,5 +144,30 @@ export const useContentTaxonomyTagsUpdater = (contentId, taxonomyId) => {
     onSettled: () => {
       queryClient.invalidateQueries({ queryKey: ['contentTaxonomyTags', contentId] });
     },
+    onSuccess: () => {
+      if (window.top != null) {
+        // Sends content tags to the parent page if is called from a iframe.
+        getContentTaxonomyTagsData(contentId).then((data) => {
+          const messageJson = {
+            contentId,
+            ...data,
+          };
+          window.top?.postMessage(
+            `[Manage tags drawer] Tags updated: ${JSON.stringify(messageJson)}`,
+            getConfig().STUDIO_BASE_URL,
+          );
+        });
+        getContentTaxonomyTagsCountData(contentId).then((data) => {
```

Comment:
It would be simpler to just send a message that the count has changed, with only the `contentId` and to let the parent frame call the API to retrieve the updated count if it wants to. That will keep the code here simpler. What do you think?

Reply:
I have also thought the same thing, but I feel that it is simpler to make and handle the API calls here and just send the data to be displayed in the template. Also I don't think there is much problem with the code here since it is temporary

Reply:
Yep, I just don't want people to copy this pattern when they move it into the MFE. If we just use React-Query in the MFE it will update without any special code, as long as we do the "query invalidation" correctly.